cmd/bosun/conf/system.go

```diff
@@ -140,6 +145,43 @@ type SMTPConf struct {
 	Password  string `json:"-"`
 }
 
+//AuthConf is configuration for bosun's authentication
+type AuthConf struct {
+	AuthEnabled bool
+	//Secret string to hash auth tokens. Needed to enable token auth.
+	TokenSecret string
+	//Secret sting used to encrypt cookie.
+	CookieSecret string
+	//LDAP configuration
+	LDAP LDAPConf
+}
+
+type LDAPConf struct {
+	// Domain name (used to make domain/username)
+	Domain string
+	// LDAP server
+	LdapAddr string
+	// allow insecure ldap connection?
+	AllowInsecure bool
+	// default permission level for anyone who can log in. Try "Reader".
```

Comment:
It comes from the string parsing in roles.go. Any permission or role name should be valid, comma seperated.